### PR TITLE
fix: handle `=` in parameters value

### PR DESCRIPTION
**Pull request by chamorin**
### Focus of the PR:
This PR focuses on improving the handling of parameters in the `applyParams` function.

### Detailed summary:
- Added a check to ensure that the parameter contains an "=" sign.
- Split the parameter into module and value based on the "=" sign.
- Updated tests to handle parameters with "=" sign and empty values.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: packages/core/src/manifest/params/apply-params.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { getModuleOrThrow } from "../../utils/get-module.js";
 
 export function applyParams(params: string[], modules: Module[]) {
   for (const param of params) {
-    const [module, value] = param.split("=", 2);
+    const [module, value] = param.split(/=(.*)/s);
     if (module === undefined || value === undefined) {
```
Comment (fubhy) [inline packages/core/src/manifest/params/apply-params.ts:7]: Let's fix this without a regex and also allow empty parameters then...

What about sth. like this

```suggestion
    const index = params.indexOf("=");
    if (index === -1) {
      throw new Error(`Invalid param ${param}. Must be in the form of "module=value" or "imported:module=value"`);
    }

    const module = param.slice(0, index);
    const value = param.slice(index + 1);
```

**Review by fubhy — CHANGES_REQUESTED** [review]
Thanks!

File: packages/core/src/manifest/params/apply-params.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { getModuleOrThrow } from "../../utils/get-module.js";
...
       throw new Error(`Invalid param ${param}. Must be in the form of "module=value" or "imported:module=value"`);
     }
```
[the remark this answers is not in the document]
Comment (chamorin) [reply packages/core/src/manifest/params/apply-params.ts]: Applied the changes and added test case for empty parameter: `mymodule=`

[the remark this answers is not in the document]
Comment (fubhy) [reply packages/core/src/manifest/params/apply-params.ts]: Great, thanks!

**Pull request merged**